### PR TITLE
draft of precipitation flux output

### PR DESCRIPTION
**Pull request by trontrytel**
A draft on how to compute precipitation flux diagnostics. I want it defined on cell faces, so that we can easily get the value at the surface.

### CODE REVIEW

File: driver/compute_diagnostics.jl [hunk 1]
```diff
@@ -298,6 +305,8 @@ function compute_diagnostics!(
                 diag_tc_f.nh_pressure_drag[k] += a_up_f[k] * aux_up_f[i].nh_pressure_drag[k] / a_up_bulk_f[k]
             end
         end
+        diag_tc_f_precip.rain_flux[k] = RB_precip(ρ_c[k] * prog_pr.q_rai[k] * aux_tc.term_vel_rain[k])
+        diag_tc_f_precip.snow_flux[k] = RB_precip(ρ_c[k] * prog_pr.q_sno[k] * aux_tc.term_vel_snow[k])
```
Comment (charleskawczynski) [inline driver/compute_diagnostics.jl:309]:
```suggestion
        diag_tc_f_precip.rain_flux[k] = RB_precip(ρ_f[k] * prog_pr.q_rai[k] * aux_tc.term_vel_rain[k])
        diag_tc_f_precip.snow_flux[k] = RB_precip(ρ_f[k] * prog_pr.q_sno[k] * aux_tc.term_vel_snow[k])
```

File: driver/compute_diagnostics.jl [hunk 2]
```diff
@@ -298,6 +305,8 @@ function compute_diagnostics!(
...
     end
```
Comment (charleskawczynski) [inline driver/compute_diagnostics.jl:310]:
```suggestion
    end
        @. diag_tc_f_precip.rain_flux = RB_precip(ρ_c * prog_pr.q_rai * aux_tc.term_vel_rain)
        @. diag_tc_f_precip.snow_flux = RB_precip(ρ_c * prog_pr.q_sno * aux_tc.term_vel_snow)
```

File: driver/compute_diagnostics.jl [hunk 3]
```diff
@@ -310,6 +317,8 @@ function compute_diagnostics!(
             end
         end
     end
+        @. diag_tc_f_precip.rain_flux = RB_precip(ρ_c * prog_pr.q_rai * aux_tc.term_vel_rain)
+        @. diag_tc_f_precip.snow_flux = RB_precip(ρ_c * prog_pr.q_sno * aux_tc.term_vel_snow)
```
Comment (haakon-e) [inline driver/compute_diagnostics.jl:321]: indentation

**Review by charleskawczynski — APPROVED** [review]

**Comment by yairchn** [comment]
Bors r+

**Comment by yairchn** [comment]
Bors r-

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**